actions/list_orgs.yaml

```diff
@@ -9,11 +9,15 @@ parameters:
   user:
     type: "string"
     description: "User name."
-    required: true
+    required: false
   base_url:
     type: "string"
     description: "GitHub API url, include /api/v3 for GitHub Enterprise."
     required: false
+  token:
+    type: "string"
+    required: false
+    description: "API token for a member of the organization"
```

Comment:
Could you add comment that this is used when `user` is not specified?

Reply:
OK.  I add it.